### PR TITLE
Add 2024Q3 Azure report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: website/content/en/status/report-2024-07-2024-09/azure.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Contact: link:https://lists.freebsd.org/subscription/freebsd-cloud[freebsd-cloud Mailing List] +
+Contact: The FreeBSD Azure Release Engineering Team <releng-azure@FreeBSD.org> +
+Contact: Wei Hu <whu@FreeBSD.org> +
+Contact: Souradeep Chakrabarti <schakrabarti@microsoft.com> +
+Contact: Colin Su <yuas@microsoft.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+In this quarter, we have published the link:https://azuremarketplace.microsoft.com/marketplace/apps/thefreebsdfoundation.freebsd-13_4[13.4-RELEASE on Azure Marketplace].
+
+Souradeep Chakrabarti from Microsoft has added feature to use hypercalls for TLB shootdown on Hyper-V and Azure.
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/azure.adoc:17]: s/added/added a/

File: website/content/en/status/report-2024-07-2024-09/azure.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Micro-benchmark shows 30-40% performance improvement on TLB shootdown.
+This his been presented at the link:https://wiki.freebsd.org/DevSummit/202409[DevSummit 202409]
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/azure.adoc:19]: s/his/has/

File: website/content/en/status/report-2024-07-2024-09/azure.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Wei Hu root caused an issue on missing CDROM device when booting FreeBSD on the latest Azure v6 VM SKU.
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/azure.adoc:21]: s/root caused/root-caused/ (I think)

File: website/content/en/status/report-2024-07-2024-09/azure.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+V6 type only offers NVMe disks to guest OS.  He also continuous bug fixing for FreeBSD MANA NIC device.
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/azure.adoc:22]: s/continuous/continues/

Also, 1 sentence per line.

**Review by pauamma — APPROVED** [review]
Approved with these nits fixed.

**Review by pauamma — APPROVED** [review]

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**